src/views/partials/update-your-details/sole-trader-answers.njk

```diff
@@ -0,0 +1,202 @@
+<h2 class="govuk-heading-m">{{ i18n.updateDetailsHeading }}</h2>
+{% if detailsAnswersUpdated.name !== detailsAnswers.name %}
+{{ govukSummaryList({
+    rows: [
+    {
+        key: {
+          text: i18n.updateYourDetailsName,
+          classes: "govuk-!-width-one-third"
+        },
+        value: {
+          text: detailsAnswersUpdated.name,
+          classes: "govuk-!-width-one-third"
+        },
```

Comment:
As discussed, below text is missing under the field values

Details updated
Pending until submitted
and accepted